src/DynamoSandbox/DynamoSandbox.csproj

```diff
@@ -98,6 +98,7 @@
     <ProjectReference Include="..\DynamoUtilities\DynamoUtilities.csproj">
       <Project>{B5F435CB-0D8A-40B1-A4F7-5ECB3CE792A9}</Project>
       <Name>DynamoUtilities</Name>
+      <Private>False</Private>
```

Comment:
when I try to debug and to see the state of an object, Visual studio says "DynamoUtilities is loaded twice" and so refuses to show state of any object